spec/javascripts/actions/story_spec.js

```diff
@@ -73,3 +73,36 @@ describe('updateStory', () => {
     });
   });
 });
+
+describe('deleteStory', () => {
```

Comment:
Should we have tests for failure cases?

Reply:
Yes, but since we need a way to dispatch actions to notify users on errors, we gonna test this later.